File: finah-desktop/src/finah_desktop/Postcode.java
package finah_desktop;

public class Postcode {
	private int Id;
	private int Postnr;
	private String Gemeente;

	public Postcode() {

	}

	public Postcode(int postcode, String gemeente) {
		this.Postnr = postcode;
		this.Gemeente = gemeente;
	}

	public int getId() {
		return Id;
	}

	public void setId(int id) {
		Id = id;
	}

	public int getPostnr() {
		return Postnr;
	}

	public void setPostnr(int postnr) {
		Postnr = postnr;
	}

	public String getGemeente() {
		return Gemeente;
	}

	public void setGemeente(String gemeente) {
		Gemeente = gemeente;
	}


}

File: finah-web/PHP/Models/Postcode.php
<?php

    /**
     * Created by PhpStorm.
     * User: Gert
     * Date: 24/03/2015
     * Time: 15:57
     */
    class Postcode
    {
        public $Id;
        public $Postnr;
        public $Gemeente;

        public function getId()
        {
            return $this->Id;
        }

        public function setId($Id)
        {
            $this->Id = $Id;
        }

        public function getPostnr()
        {
            return $this->Postnr;
        }

        public function setPostnr($Postnr)
        {
            $this->Postnr = $Postnr;
        }

        public function getGemeente()
        {
            return $this->Gemeente;
        }

        public function setGemeente($Gemeente)
        {
            $this->Gemeente = $Gemeente;
        }


    }